.github/workflows/test_chart.yaml

```diff
@@ -77,7 +77,13 @@ jobs:
         run: helm install rancher rancher-stable/rancher --namespace cattle-system --create-namespace --set bootstrapPassword=rancheradmin --set replicas=1 --set hostname="e2e.dev.rancher" --set 'extraEnv[0].name=CATTLE_FEATURES' --set global.cattle.psp.enabled=false --version ${{ env.RANCHER_VERSION }} --wait
 
       - name: Run chart-testing (install)
-        run: helm install rancher-turtles out/charts/rancher-turtles/ -n rancher-turtles-system --create-namespace --wait --set cluster-api-operator.cluster-api.enabled=false --set cluster-api-operator.enabled=false
+        run: helm install rancher-turtles out/charts/rancher-turtles/ -n rancher-turtles-system --create-namespace --wait --debug
+
+      - name: Wait for core provider rollout
+        run: sleep 30 && kubectl rollout status deployment capi-controller-manager -n capi-system --timeout=180s
 
       - name: Run chart-testing (un-install)
-        run: helm uninstall rancher-turtles -n rancher-turtles-system --wait
+        run: helm uninstall rancher-turtles -n rancher-turtles-system --cascade foreground --wait --debug
+
+      - name: Run chart re-install
+        run: helm install rancher-turtles out/charts/rancher-turtles/ -n rancher-turtles-system --create-namespace --wait --debug
```

Comment:
Would it be worth adding a check that some CAPI providers get installed after the re-install?

Reply:
I’m working on a similar e2e test change, but it will be a part of another PR. Let’s address it later?

Reply:
Sounds like a plan :+1: